### PR TITLE
Update ingest-data-iot-hub.md

### PR DESCRIPTION
**Pull request by OrrinEdenfield**
Removed the prerequisite of Visual Studio as it seems only .NET Core SDK is needed when users modify the sample code with another text editor.  I changed this to .NET Core SDK. This will make the instructions clearer & more inclusive to Linux users who can't easily run Visual Studio (would have also saved me time as the install for Visual Studio is much longer vs. the install for just the .NET Core SDK).

### CODE REVIEW

**Comment by PRMerger14** [comment]
@OrrinEdenfield : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: data-explorer/ingest-data-iot-hub.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ For general information about ingesting into Azure Data Explorer from IoT Hub, s
 * An Azure subscription. Create a [free Azure account](https://azure.microsoft.com/free/).
 * Create [a cluster and database](create-cluster-database-portal.md).
 * [A sample app](https://github.com/Azure-Samples/azure-iot-samples-csharp) and documentation for simulating a device.
-* [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) to run the sample app.
+* [.NET Core SDK](https://aka.ms/dotnet-download) to run the sample app.
```
Comment (yogilad) [inline data-explorer/ingest-data-iot-hub.md:33]:
```suggestion
* [.NET SDK] (https://dotnet.microsoft.com/download) to compile and run the sample app.
```

**Review by yogilad — CHANGES_REQUESTED** [review]
Specific suggestion in the comments

**Comment by PRMerger3** [comment]
@OrrinEdenfield : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger3** [comment]
@OrrinEdenfield : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by YaelSchuster** [comment]
Approved by ADX content team
#sign-off

Thanks, @OrrinEdenfield for your contribution!

**Comment by PRMerger12** [comment]
@YaelSchuster: I'm sorry - only the author of this article, @orspod, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by PRMerger12** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**